runtime/rasdump/javadump.cpp

```diff
@@ -1244,6 +1244,47 @@ JavaCoreDumpWriter::writeEnvironmentSection(void)
 		}
 	}
 
+#if defined(J9VM_OPT_CRIU_SUPPORT)
+	/* write restore args if they exist */
+	j9args = _VirtualMachine->checkpointState.restoreArgsList;
+	args = j9args->actualVMArgs;
+	if ((NULL != j9args) && (j9args->nOptions > 0)) {
```

Comment:
if args list is zero it should print "none". Having the  "Restore Args .." message is useful info in and of itself

Reply:
Is having the "Restore Args" message itself enough if there's no args in the list, like what's done for user args, or should we print an extra line that says "none"? Should we also change user args section to do this if so?

Reply:
We can print none. In practice user args will never be empty because of the default args.